apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -531,13 +561,11 @@ class JsDebugger extends React.Component {
              ref={debugConsole => this._debugConsole = debugConsole}
            />)}
         <div style={{display: showWatchPane ? 'initial' : 'none'}}>
-          <ProtectedStatefulDiv>
-            <div
-              id="watchersResizeBar"
-              ref={watchersResizeBar => this._watchersResizeBar = watchersResizeBar}
-              onMouseDown={this.onMouseDownWatchersResizeBar}
-            />
-          </ProtectedStatefulDiv>
+          <div
```

Comment:
Nothing outside of this component depends on a reference to `#watchersResizeBar` anymore, so we can safely remove the `ProtectedStatefulDiv`.